aten/src/ATen/native/transformers/cuda/flash_attn/fmha_api.cpp

```diff
@@ -317,21 +319,31 @@ mha_fwd(const at::Tensor &q,         // total_q x num_heads x head_size, total_q
     // We get the default generator and return the seed and offset which will
     // be used in the backward function
     auto gen = at::get_generator_or_default<at::CUDAGeneratorImpl>(c10::nullopt, at::cuda::detail::getDefaultCUDAGenerator());
-    uint64_t seed{0}, offset{0};
-    if( is_dropout ) {
-        TORCH_CHECK(at::cuda::currentStreamCaptureStatus() == at::cuda::CaptureStatus::None,
-        "scaled_dot_product_flash_attention does not support dropout with cuda graph capture mode enabled");
+    at::Tensor seed_t, offset_t;
+    if (is_dropout) {
+
         // See Note [Acquire lock when using random generators]
         std::lock_guard<std::mutex> lock(gen->mutex_);
         // generator_state = at::Tensor::wrap_tensor_impl(gen -> get_state());
```

Comment:
nit: this comment can go and was added by me when attempting to it the first go around